doc/asab/web/restapidocs.rst

```diff
@@ -0,0 +1,107 @@
+REST API Docs
+==============
+
+ASAB's API service generates a `Swagger documentation <https://swagger.io/specification>`_ which automatically shows all
+of your endpoints. However you will need to follow some steps to add summaries, tags and more.
+
+Before you start, make sure that you have a `REST API microservice <https://asab.readthedocs.io/en/latest/tutorial/web/chapter2.html>`_.
+
+If you want Authorization in Swagger, you will need a OpenIDConnect endpoint.
+
+In your microservice
+-----------
+
+First you need to initialize the API Service:
+
+.. code-block:: python
+
+    # Initialize API service
+    self.ApiService = asab.api.ApiService(self)
+
+    # Introduce Web to API Service
+    self.ApiService.initialize_web()
+
+After initializing the API Service a **/doc** endpoint will become available. You will notice
+that some or none of your endpoints have summaries, tags or descriptions.
+
+That's because you need to add a docstring to your endpoint method:
+
+.. code-block:: python
+
+    async def endpoint(self, request):
+        """
+        Summary looks like this and takes the first line from docstring.
+
+        Description of what this endpoint does.
+
+        ---
+        tags: [asab.mymicroservice]
+        """
+
+The first line of the docstring gets shown as a summary next to the endpoint in Swagger.
+There are also extra parameters you can set like tags, which work like categories.
+
+The description of the microservice also needs a docstring for it to show up:
+
+.. code-block:: python
+
+    class TutorialApp(asab.Application):
+        """
+        TutorialApp is the best microservice in the world!
+
+        <marquee>The description supports HTML tags</marquee>
+        """
+
+Authorization in Swagger
+-----------
+Authorization requires having an OpenIDConnect endpoint with an Authorization and Token endpoint (like SeaCat).
+
+First initialize the Authz service in your microservice:
```

Comment:
This page should be about swagger, not how to add authorization to your microservice - it should be in a different section. I guess it is out of the scope of your task - link to an example and add just brief comment: https://github.com/TeskaLabs/asab/blob/master/examples/web-authz-userinfo.py